CONTRIBUTING.md

```diff
@@ -88,7 +88,26 @@ docker run --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:nightly-c
 
 bazel test -c opt -k \
 --test_timeout 300,450,1200,3600 \
---test_output=errors \
+--test_output=all \
+//tensorflow_addons/...
+```
+
+### Locally Testing GPU
+```bash
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:custom-op-gpu make gpu-unit-test
+```
+
+or run manually:
+
+```bash
+docker run --runtime=nvidia --rm -it -v ${PWD}:/addons -w /addons tensorflow/tensorflow:custom-op-gpu /bin/bash
+export TF_NEED_CUDA=1
```

Comment:
TF_NEED_CUDA has existed in the custom-op-gpu image

```bash
$docker run --rm -it tensorflow/tensorflow:custom-op-gpu bash -c 'echo $TF_NEED_CUDA'
1
```

Reply:
Thanks! Removed.